mods/voidmulti.sc2mod/base.sc2data/gamedata/validatordata.xml

```diff
@@ -99,7 +99,7 @@
     </CValidatorPlayerRequirement>
     <CValidatorCombine id="HarvestableResourceandNearbyTownhall">
         <Type value="And"/>
-        <CombineArray value="HarvestableResources"/>
+        <CombineArray value="Minerals"/>
```

Comment:
Fixed an issue where MULE could be cast targeting refineries close to Command Centers

Only mineral fields will now pass.
According to Scarlett, the change originates in LotV experiments where the Mule was allowed to gather Vespene Gas as well